client/components/dot-pager/style.scss

```diff
@@ -26,41 +26,63 @@
 
 .dot-pager__controls {
 	margin: 0;
-	margin-top: 16px;
+	margin-bottom: 16px;
 	display: flex;
-	justify-content: center;
+	justify-content: flex-start;
 
 	li {
 		display: inline-flex;
 		margin: 0 4px;
 		height: 18px;
 		align-items: center;
+
+		&:first-child {
+			margin-left: 0;
+		}
+		&:last-child {
+			margin-right: 0;
+		}
+	}
+
+	.dot-pager__control-gap {
+		margin-left: auto;
 	}
 
 	button {
-		width: 12px;
-		height: 12px;
+		width: 6px;
+		height: 6px;
+		// Drawing a CSS circle
+		// stylelint-disable-next-line  declaration-property-unit-allowed-list
 		border-radius: 50%;
 		cursor: pointer;
 		background-color: var( --color-neutral-10 );
 
 		&:hover {
-			background-color: var( --color-neutral-light );
+			background-color: var( --studio-gray-20 );
 		}
 
 		&:disabled {
 			cursor: default;
 		}
 
 		&.dot-pager__control-current {
-			background-color: var( --color-primary );
+			background-color: var( --studio-gray-80 );
+		}
+
+		&.dot-pager__control-prev {
+			margin-left: auto;
```

Comment:
What's the goal for this one? Is the goal different from `.dot-pager__control-gap` ?

Reply:
Good point, I don't need two `margin-left` rules. Must be something left behind from when I was trying things out. I think I'll remove the `control-gap` class.

Reply:
I actually had to keep the `control-gap` rule and remove the `control-prev` rule, because the margin needs to be on the `<li>` element, not the `<button>`.